pkg/wolfssl/include/user_settings.h

```diff
@@ -0,0 +1,308 @@
+/* user_settings.h : custom configuration for wolfcrypt/wolfSSL */
+
+#ifndef USER_SETTINGS_H
+#define USER_SETTINGS_H
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/* System */
+#ifndef WOLFSSL_RIOT_OS
+#   define WOLFSSL_RIOT_OS 1
+#endif
+#include <random.h>
+#define CUSTOM_RAND_GENERATE random_uint32
+#define CUSTOM_RAND_TYPE uint32_t
+#define NO_WRITEV
+#define NO_DEV_RANDOM
+#define NO_FILESYSTEM
+#define NO_WOLFSSL_MEMORY
+#define NO_MAIN_DRIVER
+#define NO_SIG_WRAPPER
+#define NO_OLD_RNGNAME
+#define WOLFSSL_SMALL_STACK
+
+/* Single precision math */
+#define WOLFSSL_SP_MATH
+#define WOLFSSL_SP_SMALL
+#define SP_WORD_SIZE 32
+
+
+#ifndef MODULE_WOLFSSL_SOCKET
+#   define WOLFSSL_GNRC
+#   define WOLFSSL_USER_IO
+#endif
+
+#ifndef MODULE_WOLFSSL_TLS
+#   ifndef MODULE_WOLFSSL_TLS13
+#       define WOLFCRYPT_ONLY
+#   else
+#       define NO_OLD_TLS
+#   endif
+#endif
+
+#ifdef BOARD_NATIVE
+//#   define WOLFSSL_GENERAL_ALIGNMENT 8
+#else
+#   define WOLFSSL_GENERAL_ALIGNMENT 4
+#   ifdef CPU_ARM
+#      define TFM_ARM
```

Comment:
I don't know what this actually does, but would this be still valid for non-ARM CPUs? (same for the alignment)

Reply:
The port is 32-bit only. The ARM specific flags are used to enable platform-specific optimizations, only if available (i.e. when CPU_ARM is defined globally)

Reply:
I see. So we should exclude all non-32 bit platforms anyway? And for native it is also intended that no value is provided? (asking because the define for native is commented out)